crates/ruma-client-api/src/r0/account.rs

```diff
@@ -7,6 +7,9 @@ pub mod deactivate;
 pub mod delete_3pid;
 pub mod get_username_availability;
 pub mod register;
+#[cfg(feature = "unstable-pre-spec")]
+#[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
```

Comment:
There was a Rust change a little while ago that made these attributes redundant :)

```suggestion
```